server/plugin/command.go

```diff
@@ -702,6 +702,107 @@ func (p *Plugin) handleIssue(_ *plugin.Context, args *model.CommandArgs, paramet
 	}
 }
 
+func (p *Plugin) handleDefaultRepo(c *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
+	if len(parameters) == 0 {
+		return "Invalid issue command. Available command is 'set', 'get' and 'unset'."
+	}
+
+	command := parameters[0]
+	parameters = parameters[1:]
+
+	switch {
+	case command == "set":
+		return p.handleSetDefaultRepo(c, args, parameters, userInfo)
+	case command == "get":
+		return p.handleGetDefaultRepo(c, args, parameters, userInfo)
+	case command == "unset":
+		return p.handleUnSetDefaultRepo(c, args, parameters, userInfo)
+	default:
+		return fmt.Sprintf("Unknown subcommand %v", command)
+	}
+}
+
+func (p *Plugin) handleSetDefaultRepo(_ *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
+	if len(parameters) == 0 {
+		return "Please specify a repository."
+	}
+
+	repo := parameters[0]
+	config := p.getConfiguration()
+	baseURL := config.getBaseURL()
+	owner, repo := parseOwnerAndRepo(repo, baseURL)
+	if owner == "" && repo == "" {
+		return "invalid repository"
+	}
+
+	owner = strings.ToLower(owner)
+	repo = strings.ToLower(repo)
+
+	if config.GitHubOrg != "" && strings.ToLower(config.GitHubOrg) != owner {
+		return "repository is not part of the locked github organization"
```

Comment:
```suggestion
		return "Repository is not part of the locked Github organization"
```
We should also add the locked org name here